packages/beacon-node/src/execution/engine/utils.ts

```diff
@@ -30,15 +32,10 @@ export class ExecutionEngineMockJsonRpcClient implements IJsonRpcHttpClient {
   }
 }
 
-const fatalErrorCodes = ["ECONNREFUSED", "ENOTFOUND", "EAI_AGAIN"];
-const connectionErrorCodes = ["ECONNRESET", "ECONNABORTED"];
+export const HTTP_FATAL_ERROR_CODES = ["ECONNREFUSED", "ENOTFOUND", "EAI_AGAIN"];
+export const HTTP_CONNECTION_ERROR_CODES = ["ECONNRESET", "ECONNABORTED"];
 
-export function getExecutionEngineState({
-  payloadError,
-  payloadStatus,
-}:
-  | {payloadStatus: ExecutePayloadStatus; payloadError?: never}
-  | {payloadStatus?: never; payloadError: unknown}): ExecutionEngineState {
+function getExecutionEngineStateForPayloadStatus(payloadStatus: ExecutePayloadStatus): ExecutionEngineState {
   switch (payloadStatus) {
     case ExecutePayloadStatus.ACCEPTED:
```

Comment:
```suggestion
    case ExecPayloadStatus.ACCEPTED:
```
either we rename to this `ExecPayloadStatus` or to `ExecutionPayloadStatus`

Reply:
Renamed to `ExecutionPayloadStatus`.